gematria/testing/python/ir_utils.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+import os
+import pandas
+import pyarrow
+from pyarrow import parquet
+from rules_python.python.runfiles import runfiles
+
+
+def _get_llvm_binary_path(tool_name: str) -> str:
+  runfiles_env = runfiles.Create(os.environ)
+  assert runfiles_env is not None
+  return runfiles_env.Rlocation('llvm-project/llvm/' + tool_name)
+
+
+def get_bc_from_ir(ir: str) -> bytes:
+  """Generates bitcode for an IR string.
+  Takes an IR string as input and outputs the bitcode representation of the
+  input textual IR as bytes.
+  Args:
```

Comment:
Very nit: Add an empty line between the first line and then before each section (Args, Returns, Raises).

Same for the other comments.

See https://google.github.io/styleguide/pyguide.html#381-docstrings and https://google.github.io/styleguide/pyguide.html#383-functions-and-methods for more details.